CHANGELOG.adoc

```diff
@@ -1351,5 +1351,9 @@ Ignore PRs:
 |===
 | | Description | PR
 
+| 🎁
+| Add `env-value-from` flag & keep order of env vars in created/updated Services
+| https://github.com/knative/client/pull/1328[#1328]
+
```

Comment:
This should go to the top of the changelog. The latest `main` should have the initialized `Unreleased` section now.